lib/std/zig/cross_target.zig

```diff
@@ -694,6 +718,14 @@ pub const CrossTarget = struct {
         set.removeFeatureSet(self.cpu_features_sub);
     }
 
+    /// This is a dumb parser to essentially validate that the vendor provided is one Clang will accept.
+    fn parseVendor(result: *CrossTarget, diags: *ParseOptions.Diagnostics, text: []const u8) !void {
+        result.vendor_tag = std.meta.stringToEnum(Target.Vendor, text) orelse
+            return error.UnknownVendor;
```

Comment:
Do I need to add this to the enum defined [here](https://github.com/ziglang/zig/blob/30d01c8fea68baab396081da040f49defa494088/src/stage1.zig#L182-L264)? It is labeled ABI which makes me think it would also need to be added [here](https://github.com/ziglang/zig/blob/c12704a3395a11c28569ebb160e6ac33091d983a/src/stage1/error.cpp#L11-L93) and [here](https://github.com/ziglang/zig/blob/c12704a3395a11c28569ebb160e6ac33091d983a/src/stage1/stage2.h#L35-L117). It compiled and ran the tests without doing so, so I didn't worry about it for the initial commit.